internal/capability/auditor.go

```diff
@@ -215,10 +205,10 @@ func (ca *CapAuditor) RunAudits(ctx context.Context, fs afero.Fs, reportWriter i
 				withNamespace(audit.namespace),
 				withOperatorGroupData(&audit.operatorGroupData),
 				withSubscription(&audit.subscription),
-				withTimeout(int(audit.csvWaitTime)),
+				withTimeout(options.timeout),
 				withCustomResources(audit.customResources),
-				withFilesystem(fs),
-				withReportWriter(reportWriter),
+				withFilesystem(options.fs),
+				withReportWriter(options.reportWriter),
 			)
 			if auditFn == nil {
 				logger.Errorf("invalid audit plan specified: %s", function)
```

Comment:
I know this is not part of the change but  somehow this is happening:

{"level":"error","ts":1666986003.459054,"caller":"logger/logger.go:62","message":"invalid audit plan specified: OperatorCleanUp","stacktrace":"github.com/opdev/opcap/internal/logger.Errorf\n\t/home/alex/go/src/github.com/acmenezes/opcap/internal/logger/logger.go:62\ngithub.com/opdev/opcap/internal/capability.RunAudits\n\t/home/alex/go/src/github.com/acmenezes/opcap/internal/capability/auditor.go:214\ngithub.com/opdev/opcap/cmd.checkCmd.func1\n\t/home/alex/go/src/github.com/acmenezes/opcap/cmd/check.go:49\ngithub.com/spf13/cobra.(*Command).execute\n\t/home/alex/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:856\ngithub.com/spf13/cobra.(*Command).ExecuteC\n\t/home/alex/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:974\ngithub.com/spf13/cobra.(*Command).Execute\n\t/home/alex/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:902\ngithub.com/spf13/cobra.(*Command).ExecuteContext\n\t/home/alex/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:895\ngithub.com/opdev/opcap/cmd.Execute\n\t/home/alex/go/src/github.com/acmenezes/opcap/cmd/root.go:44\nmain.main\n\t/home/alex/go/src/github.com/acmenezes/opcap/main.go:17\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:250"}

Reply:
I just ran `operator check` without flags. Was just testing with minimal input and the error comes up. Does it happen to you too?

Reply:
Thanks!